packages/flutter_image/lib/network.dart

```diff
@@ -53,6 +50,33 @@ class NetworkImageWithRetry extends ImageProvider<NetworkImageWithRetry> {
   /// fetch attempts failed.
   final FetchStrategy fetchStrategy;
 
+  /// HTTP Headers to add to the request.
+  ///
+  /// Keys from this map will be used as the header field names and the
+  /// values will be used as the header values. A list of header names can
+  /// be found at https://datatracker.ietf.org/doc/html/rfc7231#section-8.3
+  ///
+  /// Some headers are single valued, and for these, adding a value will
```

Comment:
This sentence doesn't make sense in the context of this API, which isn't an `add`.

Reply:
Removed this sentence.